common/src/main/java/com/wynntils/features/user/FilterAdminCommandsFeature.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.brigadier.tree.CommandNode;
+import com.mojang.brigadier.tree.RootCommandNode;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.CommandsPacketEvent;
+import java.util.Arrays;
+import java.util.List;
+import net.minecraft.commands.SharedSuggestionProvider;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class FilterAdminCommandsFeature extends UserFeature {
+    private static final List<String> FILTERED_COMMANDS = Arrays.asList(new String[] {
+        "bungee",
+        "connect",
+        "galert",
+        "gcountdown",
+        "glist",
+        "gsend",
+        "perms",
+        "pfind",
+        "plist",
+        "pwlist",
+        "sendtoall",
+        "servers",
+        "sparkb",
+        "sparkbungee",
+        "wcl",
+        "wynnproxy"
+    });
+
+    @SubscribeEvent
+    public void onCommandPacket(CommandsPacketEvent event) {
+        RootCommandNode<SharedSuggestionProvider> root = event.getRoot();
+
+        RootCommandNode<SharedSuggestionProvider> newRoot = new RootCommandNode<>();
+        for (CommandNode<SharedSuggestionProvider> child : root.getChildren()) {
+            if (!FILTERED_COMMANDS.contains(child.getName())) {
+                newRoot.addChild(child);
+            }
+        }
+
+        // We also need to add the arguments for all commands
+        newRoot.addChild(root.getChild("args"));
```

Comment:
What does this do?

Reply:
All command literals have a hardcoded dependency on an "args" string argument node. I assume this is something setup by whatever mod they have to inject commands like this (bungee, perhaps). 

In theory, I should recurse over all child nodes and see if they have more argument types to be added. At least, that's what I assume I'm supposed to do. But I could not be bothered. Should Wynncraft drastically change how their admin commands are provided, then the user need to disable this feature until we can fix it.

Reply:
(keep in mind this is only command suggestions, so even if we filter out something we should not, the user doesn't lose much.

Reply:
It seemed like I could remove it without any bad effects. I don't fully understand the Brigadier structure. But if it works without it, I'll skip it.